.envrc.local.default

```diff
@@ -0,0 +1,5 @@
+# In order to run integration tests
+# we need ngin(xz) locally built using nix.
+# Copy/move this file to .envrc.local for that to work
+export COMPILE_NGINX_USING_NIX=1
+export INTEGRATION_USE_NGINZ=1
```

Comment:
Instead of this, let's just change the default behaviour when `COMPILE_NGINX_USING_NIX` and `INTEGRATION_USE_NGINZ` are not defined.

Reply:
I checked: That's the only two variables I have set in my .envrc.local file

Reply:
Sounds like a good idea, but I'm not that versed in bash, I wouldn't want to break anything fiddling with that.

Reply:
I also had to figure it out (and will forget immediately again :)